.github/scripts/testing-test-results/libatomic.sum

```diff
@@ -0,0 +1,71 @@
+Test run by root on Mon Aug  5 21:31:30 2024
```

Comment:
I am not sure it should be committed.

Reply:
I'm using them for debugging here, so I don't need to wait for pipeline to build whole toolchain and run all tests. I'm having a problem that I can't access `$GITHUB_STEP_SUMMARY` variable in the `.yml` file.

: https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/runs/10575138154/job/29298199546

![image](https://github.com/user-attachments/assets/43c34a63-83ed-4737-bb04-da50cac3601e)

Reply:
I see now, I'll need to print the top10 summary to a temp file . And then `echo` the temp file to `GITHUB_STEP_SUMMARY` in a separate step in bash outside WSL. Similarly how it's done in `Create summary` and `Present summary` steps

Reply:
That's one option. Other option might be to add `GITHUB_STEP_SUMMARY` to `WSLENV` variable but I am not sure this actually works.

Reply:
Added `GITHUB_STEP_SUMMARY` to `WSLENV` variable to keep everything in one step and one shell. `GITHUB_STEP_SUMMARY` was already a WSL path and writing to it worked as expected in my tests but I'm waiting for complete pipeline to finish to be 100% sure: https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/runs/10590901375